pipeline/recon-pipeline.py

```diff
@@ -235,6 +235,16 @@ def do_scan(self, args):
         # luigi --module pipeline.recon.web.webanalyze WebanalyzeScan --target-file tesla --top-ports 1000 --interface eth0
         command = ["luigi", "--module", scans.get(args.scantype)[0]]
 
+        if args.target and args.target_file:
+            self.async_alert(style(f"[!] You can't use the --target and --target-file options in the same scan!"))
+            return
```

Comment:
mutual exclusion should be enforced through [argparse's add_mutually_exclusive_group](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group) when possible.

Also, either target-file or target is required, so `add_mutually_exclusive_group(required=True)` would be appropriate.  

Check [line 60 in recon.parsers](https://github.com/epi052/recon-pipeline/blob/store-results-in-a-database-project/pipeline/recon/parsers.py#L60) for an example.